quiche/include/quiche.h

```diff
@@ -768,6 +768,11 @@ typedef struct {
     size_t value_len;
 } quiche_h3_header;
 
+typedef struct {
```

Comment:
Need to add the description from the Rust struct here.

Reply:
done